linalg/hypre.hpp

```diff
@@ -989,9 +1034,83 @@ class HypreBoomerAMG : public HypreSolver
        As with SetSystemsOptions(), this solver assumes Ordering::byVDIM. */
    void SetElasticityOptions(ParFiniteElementSpace *fespace);
 
+#if MFEM_HYPRE_VERSION >= 21800
+   /* distance parameter takes on values {1,2,15} for lAIR, meaning R is built using
+   distance 1 neighbors, distance two neighbors, or distance two on processor and
+   distance 1 off processor (i.e., distance 1.5 --> 15).        */
```

Comment:
Some documentation (or at least references to relevant hypre documentation) is badly needed here. There are a lot of options, and it's not clear to the non-expert what they do.